src/main/java/net/dv8tion/jda/internal/entities/MemberImpl.java

```diff
@@ -116,6 +116,12 @@ public OffsetDateTime getTimeBoosted()
         return boostDate != 0 ? Helpers.toOffset(boostDate) : null;
```

Comment:
```suggestion
        return isBoosting() ? Helpers.toOffset(boostDate) : null;
```
This is better IMO, would like to hear your opinions on this.

Reply:
Hmm, I'm not sure about this, both approaches are certainly valid, it reuses the new code but I feel it adds an unnecessary step in the stack.

Reply:
If we decide to change what `boostDate` contains if the member is not boosting, then this will be easier. Also, it is also more readable
About the extra method call, I think it should be fine, I doubt we should try to over optimize our code like that.

Reply:
So @SweetSofiMC , are you going to implement this suggestion?

Reply:
Yes, it shouldn't be much of a performance problem. Will change it!